exporter.py

```diff
@@ -26,37 +38,44 @@ def parse_params():
     parser.add_option("-r", "--rubric", help="metric rubric only for VMStatsCollector", action="store", dest="rubric")
     (options, args) = parser.parse_args()
 
-
     if options.inventory:
         os.environ['INVENTORY'] = options.inventory
+    if options.info:
+        logger.setLevel(logging.INFO)
+        ConsoleHandler.setLevel(logging.INFO)
+        logger.info(f'Starting exporter logging on INFO level')
     if options.debug:
-        os.environ['DEBUG'] = "1"
-        print('DEBUG enabled')
-    else:
-        if 'DEBUG' not in os.environ.keys():
-            os.environ['DEBUG'] = "0"
-        else:
-            print('DEBUG enabled')
+        logger.setLevel(logging.DEBUG)
+        ConsoleHandler.setLevel(logging.DEBUG)
+        logger.debug(f'Starting exporter logging on DEBUG level')
+    if not options.debug and not options.info:
+        logger.setLevel(logging.WARNING)
+        ConsoleHandler.setLevel(logging.WARNING)
+        logger.warning(f'Starting exporter logging on WARNING, ERROR and CRITICAL level')
     if options.port:
         os.environ['PORT'] = options.port
     if options.config:
         os.environ['CONFIG'] = options.config
     if not options.collectors:
+        logger.debug(f'Exporter using default collectors from config')
         options.collectors = default_collectors()
     if options.target:
         os.environ['TARGET'] = options.target
+    if not options.target:
+        target = get_targets()[0]
```

Comment:
that means we are always defaulting to taget zero?

Reply:
Not always. Just in case there is no target given in the parameters. I would like to make sure that the exporter will not crash without a target. This could be annoying for tests. When live, this is given anyway.